### ISSUE TITLE
Select development environment

### ISSUE THREAD

**Issue by bvacaliuc** [uvdl/meta-ornl#1]
Would like to determine the appropriate build environment and distribution generation that provides:
 * minimal required OS (packages and dependencies)
 * package management (e.g. apt-get) that works well
 * supports suspend to memory (i.e. it isnt broken)
 * supports i.MX6 special functions, VPU/GPU coding and gstreamer

**Comment by edisonf** [comment]
@bvacaliuc 

We will be using yocto. All instructions and SW will be found [here](https://github.com/uvdl/yocto-ornl/tree/develop)

**Comment by bvacaliuc** [comment]
Just so you know, I followed the instructions in the [feature/ornl-image-support](https://github.com/uvdl/yocto-ornl/blob/feature/ornl-image-support/README.md) branch and while it produced an SD card, when I booted the system to SD card, it failed to even start the console and uboot.

I diagnosed it to a missing bootloader (first 8192 blocks of the SD card), so that needs to be added to recipes.  I transferred the first 8192 blocks from the debian SD card and got the Yocto SD card to boot.

The base Yocto image is pretty rough.  No wifi device, appears to be missing a package manager, and appears to be missing the parts discussing [Yocto NAND Flash Burning](http://variwiki.com/index.php?title=Yocto_NAND_Flash_Burning&release=RELEASE_SUMO_V1.1_VAR-SOM-MX6).  I also tested suspend to mem and while it did suspend, I was not able to get the DT6CustomBoard to wake up by any of the typical means.

 I realize you might still be working on this, but just so you are all very clear of my expectations.  It looks like this Yocto path requires work to get to the functionality that Debian *started* with.
